assets/js/buttons-examples.js

```diff
@@ -126,6 +126,11 @@ $(document).ready(function () {
           label: "Splash",
           classes: "splash-btn",
         },
+        {
```

Comment:
Looks cute but my only remark is to place this button in the correct alphabetical order under "Animated" (sorry I didn't notice this before).

Reply:
No problem. Should have spotted that